### DISCUSSION TITLE
no recovery mod

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13101] [Ideas]
how the mod works that any hp recovery is negated and will not happen, hp loss still happens at normal expect that hp drain will not happen

this can be used to aim for better scores even with misses

**Comment by ThatStella7922** [comment]
what im getting from this is that:
- passive HP drain is removed
- HP loss for missing is normal
- remove all HP recovery

if this is what you mean, then this is a very cool idea

**Comment by smoogipoo** [comment]
Should be pretty quick implementation through adding a method `AdjustHealthIncrease` to 
`IApplicableToHealthProcessor`, and then using it inside `HealthProcessor`. `IApplicableToScoreProcessor` has a similar implementation for `AdjustRank`.

**Comment by vmaggioli** [comment]
I've picked this up and am currently working on it, just a bit confused how I can create a new Icon for the mod as I only see hex address references in `OsuIcon.cs`

**Comment by peppy** [comment]
For the time being you cannot. Choose one which fits or let it fall back to the acronym.

**Comment by peppy** [comment]
To elaborate, icons are currently pulled from a font store, which isn't the best place for them (adding new ones comes with an overhead of adding to a ttf/otf file). Going forward we'll want to move these out to their own sprite resources.
